docs/azure_jumpstart_ag/contoso_supermarket/k8s_infra_observability/_index.md

```diff
@@ -4,4 +4,121 @@ weight: 100
+
+## Access the dashboards
+
+As an Operations team member at Contoso Supermarket, you will have access to the Grafana dashboards for infrastructure observability. These dashboards provide a comprehensive view of the health, performance, and metrics of the cloud to edge infrastructure. To access the Grafana dashboards follow the below steps.
+
+- Connect to the Client VM `Ag-VM-Client` using the instructions in the [Deployment Guide](https://github.com/microsoft/azure_arc/blob/jumpstart_ag/docs/azure_jumpstart_ag/contoso_supermarket/deployment/_index.md#connecting-to-the-agora-client-virtual-machine).
+
+### Prod Stores
+
+- Open the Edge browser, expand Grafana in the Favorites Bar and select `Grafana Prod`.
+
+    ![Observability grafana_prod diagram](./img/grafana_prod.png)
+
+- Login using the Contoso Operator username `arcdemo` and the Windows Admin Password you provided when you created your deployment.
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_login.png)
+
+- Click `Chicago - Kubernetes / Views / Global` dashboard to review the overall health of the cluster.
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_dashboard_global.png)
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_db_global01.png)
+
+- Review different panels in the dashboard to see the metrics collected from the Kubernetes cluster.
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_db_global02.png)
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_db_global03.png)
+
+- Click `Home` to go back to the home page and review the other dashboards available.
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_dashboard_home.png)
+
+- Click `Chicago - Node Exporter Full` dashboard to review the Chicago store node metrics.
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_dashboard_node.png)
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_db_node01.png)
+
+- Review different panels in the dashboard to see the metrics collected from the Kubernetes cluster.
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_db_node02.png)
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_db_node03.png)
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_db_node04.png)
+
+- Repeat the same steps to access the `Seattle Store` dashboards and review the cluster and node metrics.
+
+    ![Observability grafana_prod diagram](./img/grafana_prod_dashboard_seattle.png)
+
+### Staging Store
+
+- Open the Edge browser, expand Grafana in the Favorites Bar and select `Grafana Staging`.
+
+    ![Observability grafana_prod diagram](./img/grafana_staging.png)
+
+- Login using the Contoso Operator username `arcdemo` and the Windows Admin Password you provided when you created your deployment. Review the available dashboards.
```

Comment:
arcdemo?

Reply:
Login using the Windows Admin Username and Password you provided when you created your deployment.